driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -686,17 +652,7 @@ public void onException(Connection connection, Exception exception, long latency
 
                 @Override
                 public boolean onTimeout(Connection connection, long latency, int retryCount) {
-                    QueryState queryState = queryStateRef.get();
```

Comment:
This was more or less identical to the main `onTimeout` implementation, so now that it goes through the retry policy I thought best to just call the main method here.